src/components/MapImageSlider.vue

```diff
@@ -14,7 +14,7 @@
         id="cartogram-svg"
         xmlns="http://www.w3.org/2000/svg"
         xmlns:xlink="http://www.w3.org/1999/xlink"
-        viewBox="0 -50 770 420"
+        viewBox="0 -50 502 405"
```

Comment:
when we rebuild the SVG, this change will get wiped out

Reply:
Can you fix here, too? https://github.com/usgs-makerspace/gages-through-the-ages/blob/master/data_processing_pipeline/6_visualize.yml#L23-L26

Reply:
I don't think you need to rebuild it, but just so that the next time someone does, it is fixed.

Reply:
updated